src/targets/browser/index.jsx

```diff
@@ -68,7 +72,9 @@ document.addEventListener('DOMContentLoaded', () => {
 
   const protocol = window.location ? window.location.protocol : 'https:'
 
-  const { isPublic, token } = getToken(data)
+  const shareCode = getPublicSharecode()
+  const token = shareCode || data.cozyToken
+  const isPublic = shareCode || !token || token == ''
```

Comment:
Why don't we have a separate entry point like in other apps? I think I already asked the question but don't have an answer

Reply:
We could. There is no better answer that « this is about the exact same code with the same bundle, so I did not see any reason to have a separate bundle »

Reply:
There is no other protection from stack POV when we add a public true in the manifest?

Reply:
nope, it only deactivate the need of being authenticated and does not generate a token in the HTML (but uses the one front the `sharecode` parameter if there is one)

The bundle is public anyways, and does not include any private data